proto/def/contract.proto

```diff
@@ -43,6 +43,12 @@ message EgressConfig {
   uint64 backoffDelay = 4;
 }
 
+// Check dev.knative.eventing.kafka.broker.dispatcher.consumer.DeliveryGuarantee for more details
+enum DeliveryGuarantee {
+  ORDERED = 0;
+  UNORDERED = 1;
+}
```

Comment:
Since we will probably add `at least once` / `at most once`, can you rename this enum to something less ambiguous?

Reply:
Why should we have at most once? Don't we always support at least once and that's it? Do you plan to have at most once?

Reply:
Yes, see https://github.com/knative-sandbox/eventing-kafka/issues/413

Reply:
At least, that's the plan / there is a possibility.

Since we will use the annotation suffix `delivery.order`, I'd call this `DeliveryOrder` to keep the language or mapping consistent and straightforward.

Reply:
Ok I've renamed both enums to DeliveryOrder